networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/nsxv3_facada.py

```diff
@@ -612,6 +607,29 @@ def update_security_group_capabilities(self,
 
         self.update(sdk_service=Sections, sdk_model=fs)
 
+    def update_security_group_revision_number(self, sdk_model):
+        """
+        Sets default revision_number if such is not already set
+        """
+        default_tag = Tag(scope=nsxv3_constants.NSXV3_REVISION_SCOPE, tag="-1")
+
+        sdk_model.tags = sdk_model.tags if sdk_model.tags else []
+        revision = None
+        for tag in sdk_model.tags:
+            if tag.scope == default_tag.scope:
+                revision = tag.tag
+                break
+        if revision is None or not sdk_model.tags:
```

Comment:
I don't think the condition after the `or` makes sense with above's loop. Revision will always be `None` if there are no tags. So I'd remove it to have less code to confuse people.